### PR TITLE
Add clarification for factories.yml, refs #11030

### PR DESCRIPTION
**Pull request by MikeFE**
Point user to proper factories.yml file to override settings in.

### CODE REVIEW

File: admin-manual/maintenance/logging.rst [hunk 1]
```diff
@@ -41,14 +41,17 @@ The ``factories.yml`` file in AtoM has 4 main environmentally-aware
...
+within each and overriden. According to Symfony:
 
     When symfony needs a value from a configuration file, it merges the
     configuration found in the current environment section with the ``all``
     configuration. The special ``all`` section describes the default
     configuration for all environments. If the environment section is not
     defined, symfony falls back to the ``all`` configuration.
 
+If you are an administrator and wish to override factories.yml settings, it's
+recommended you modify apps/qubit/config/factories.yml.
```
Comment (sevein) [inline admin-manual/maintenance/logging.rst]: If you wrap the path with ``` ``double backticks`` ``` you get that nice formatting when it's rendered as HTML.

PS: It wasn't easy to escape to backticks in GitHub, haha

**Comment by MikeFE** [comment]
Ok added

**Review by sevein — APPROVED** [review]

**Comment by fiver-watson** [comment]
Hey @MikeFE I just noticed this hanging out here - looks like it was approved, but never got merged? Can you double-check it, and if it's still applicable, consider merging it to 2.4 and 2.5?

File: admin-manual/maintenance/logging.rst [hunk 2]
```diff
@@ -41,14 +41,17 @@ The ``factories.yml`` file in AtoM has 4 main environmentally-aware
 ``dev`` (for development and debugging - i.e. ``qubit_dev.php``, when the site
 is in :ref:`debug-mode`), and ``all``, which are  the default settings
 inherited by the other factories, unless they are more explicitly defined
-within each and overrriden. According to Symfony:
+within each and overriden. According to Symfony:
```
Comment (fiver-watson) [inline admin-manual/maintenance/logging.rst]: also shouldn't this be overridden?

**Review by anvit — APPROVED** [review]
LGTM!

**Pull request merged**